pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -46,8 +46,10 @@ import (
 
 const (
 	// loopPeriod is the amount of time the reconciler loop waits between
-	// successive executions
-	reconcilerLoopPeriod time.Duration = 100 * time.Millisecond
+	// successive executions. This has been increased from every 100 ms to
+	// 1 minute since the timing has created an enormous amount of API traffic on
+	// such clouds as AWS.
+	reconcilerLoopPeriod time.Duration = 1 * time.Minute
```

Comment:
This doesn't increase or decrease the rate of API calls, and break a lot of things (by reducing the rate at which the attach detach controller can react, so let's leave this alone.